packages/reakit/src/Toolbar/__examples__/ToolbarWithMenu/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { RefObject } from "react";
+import {
+  useToolbarState,
+  Toolbar,
+  ToolbarItem,
+  ToolbarProps,
+} from "reakit/Toolbar";
+import { useMenuState, MenuButton, Menu, MenuItem } from "reakit/Menu";
+import { Button } from "reakit/Button";
+
+const MoreItems = React.forwardRef((props: ToolbarProps, ref) => {
```

Comment:
Didn't test it, but I think you don't need to explicitly cast `ref` by doing this:
```suggestion
const MoreItems = React.forwardRef<HTMLButtonElement, ToolbarProps>((props, ref) => {